.ci/azure/linux.yml

```diff
@@ -455,30 +455,39 @@ jobs:
     displayName: 'TensorFlow 1 Layer Tests - TF FE'
     continueOnError: false
 
-  - script: |
-      python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
-      export PYTHONPATH=$(LAYER_TESTS_DIR):$PYTHONPATH
-      $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/tensorflow_tests/test_tf_Roll.py --ir_version=10 --junitxml=$(INSTALL_TEST_DIR)/TEST-tf_Roll.xmlTEST
-    displayName: 'TensorFlow 1 Layer Tests - Legacy FE'
-    continueOnError: false
-
-  - script: |
-      python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
-      export PYTHONPATH=$(LAYER_TESTS_DIR):$PYTHONPATH
-      export TEST_DEVICE=CPU
-      $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/tensorflow2_keras_tests/test_tf2_keras_activation.py --ir_version=11 --junitxml=./TEST-tf2_Activation.xmlTEST -k "sigmoid"
-    displayName: 'TensorFlow 2 Layer Tests - Legacy FE'
-    continueOnError: false
-
-  - task: PublishTestResults@2
-    condition: always()
-    inputs:
-      testResultsFormat: 'JUnit' # Options: JUnit, NUnit, VSTest, xUnit, cTest
-      testResultsFiles: '**/TEST-*.xml'
-      #searchFolder: '$(BUILD_DIR)'
-      mergeTestResults: false # Optional
-      #failTaskOnFailedTests: false # Optional
-      #testRunTitle: 'Pre/Post-Commit' # Optional
-      buildPlatform: 'x64' # Optional
-      buildConfiguration: 'Linux' # Optional
-      #publishRunAttachments: true # Optional
+    - script: |
+        python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+        export PYTHONPATH=$(LAYER_TESTS_DIR):$PYTHONPATH
+        $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/tensorflow_tests/test_tf_Roll.py --ir_version=10 --junitxml=$(INSTALL_TEST_DIR)/TEST-tf_Roll.xmlTEST
+      displayName: 'TensorFlow 1 Layer Tests - Legacy FE'
+      continueOnError: false
+
+    - script: |
+        python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+        export PYTHONPATH=$(LAYER_TESTS_DIR):$PYTHONPATH
+        export TEST_DEVICE=CPU
+        $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/tensorflow2_keras_tests/test_tf2_keras_activation.py --ir_version=11 --junitxml=./TEST-tf2_Activation.xmlTEST -k "sigmoid"
+      displayName: 'TensorFlow 2 Layer Tests - Legacy FE'
+      continueOnError: false
+
+    - script: |
+        python3 -m pip install -r $(LAYER_TESTS_DIR)/requirements.txt
+        export PYTHONPATH=$(REPO_DIR)/tools/mo/:$(LAYER_TESTS_DIR):$PYTHONPATH
+        export OV_FRONTEND_PATH=$(INSTALL_DIR)/runtime/lib/intel64
+        export TEST_DEVICE=CPU
+        $(RUN_PREFIX) python3 -m pytest $(LAYER_TESTS_DIR)/tensorflow2_keras_tests/ --use_new_frontend -m precommit_tf_fe --junitxml=$(INSTALL_TEST_DIR)/TEST-tf2_keras_fe.xmlTEST
+      displayName: 'TensorFlow 2 Layer Tests - New FE'
+      continueOnError: false
```

Comment:
Please put layer tests for TF FE first and legacy after. Please align the names of these tests with the current version

Reply:
`TensorFlow 2 Layer Tests - New FE` -> `TensorFlow 2 Layer Tests - TF FE`

Reply:
Done, please, check